client/state/reader/start/selectors.js

```diff
@@ -19,6 +36,16 @@ export function getRecommendations( state ) {
 }
 
 /**
+ * Returns recommendation IDs.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array} Recommendations IDs
+ */
+export function getRecommendationIds( state ) {
+	return map( state.reader.start.items, 'ID' );
```

Comment:
additionally, if we switch to keying this map off of ID, it could just be a memoized call to `Object.keys()`.